helm/signalfx-adapter/values.yaml

```diff
@@ -79,6 +78,14 @@ spanNameAttribute: 'request.path | "/"'
 # within the tracespan instance used by the adapter.
 tracespanParams:
 
+# The expression to use in the tracespan rule that Istio uses to determine
+# whether to send it to the adapter.
+tracespanMatchQuery: '(context.protocol == "http" || context.protocol == "grpc") && request.host != "mixer"'
```

Comment:
Might want to include pilot here? Not sure which requests get classified to be sent to the handler, but pilot gets policy checks and might have the same problem as mixer?

Reply:
I don't see any calls to pilot that get reported to the adapter in my setup.  Let me know if you see anything in your cluster and I'll consider adding it to the default.